Doc/whatsnew/3.2.rst

```diff
@@ -50,7 +50,9 @@
 
 This article explains the new features in Python 3.2 as compared to 3.1.  It
 focuses on a few highlights and gives a few examples.  For full details, see the
-`Misc/NEWS <https://hg.python.org/cpython/file/3.2/Misc/NEWS>`_ file.
+`Misc/NEWS
+<https://github.com/python/cpython/blob/260a788767ff6dadd04ecf9e9e26b363ca28c8ae/Misc/NEWS>`_
+file.
```

Comment:
The original link https://hg.python.org/cpython/file/3.2/Misc/NEWS actually works (for now) and the replacement is not identical. Judging by the headings
original:
> What's New in Python 3.2.7?

replacement:
> What's New in Python 3.2?